dev/ci/README-users.md

```diff
@@ -17,10 +18,8 @@ On the condition that:
 - Your project is publicly available in a git repository and we can easily
   send patches to you (e.g. through pull / merge requests).
 
-- You react in a timely manner to discuss / integrate those patches.
-  When seeking your help for preparing such patches, we will accept
-  that it takes longer than when we are just requesting to integrate a
-  simple (and already fully prepared) patch.
+- You react in a timely manner to adapt to the few requested changes
+  required by Coq developers (in a 7 days timeframe) or to integrate their patches.
```

Comment:
```suggestion
  required by Coq developers or to integrate their patches (in 7 days for simple requests).
```

Reply:
This is changing the meaning of the policy, and we don't know who is to judge what is a simple request, so why not rather say:
```suggestion
  required by Coq developers or to integrate their patches (in a 7 days timeframe,
  unless you request more time or Coq developers give you longer, e.g., in case
  of more complex changes).
```

Reply:
I agree, I tried something.